__tests__/components/forecast-summaries.test.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import ForecastSummaries from '../../src/components/forecast-summaries';
+import ForecastSummary from '../../src/components/forecast-summary';
+
+const forecasts = [
+  {
+    date: 123,
+    description: 'date1',
+    icon: 'icon1',
+    temperature: {
+      max: 999,
+    },
+  },
+  {
+    date: 456,
+    description: 'date2',
+    icon: 'icon2',
+    temperature: {
+      max: 777,
+    },
+  },
+];
+it('renders the correct amount of ForecastSummary components', () => {
```

Comment:
You can still use the `beforeEach()` block here. Otherwise you just end up typing the same code over again.